modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -298,20 +299,18 @@ function MultipleChoice(props) {
     selectOption(...args);
   }
 
-  // display error if minimum is not met, display 'at least' if there is no maximum or if max is greater than min
-  const warning = selection.length < minAnswers && (
-    <Typography color={error ? 'error' : 'textSecondary'} className={classes.warningTypography}>
-      Please select {maxAnswers !== minAnswers && "at least"} {minAnswers} option{minAnswers > 1 && "s"}.
-    </Typography>
-    );
-
   // Remove the ["", ""] unless there are only zero or one answer items
   var answers = selection.map(item => item[VALUE_POS] === GHOST_SENTINEL ? [item[LABEL_POS], item[LABEL_POS]] : item);
   answers = ((answers.length < 2) ? answers : answers.filter(item => item[LABEL_POS] !== ''));
 
+  // When counting current answers for proper highlighting of answer instructions to the user, exclude the empty one
+  let currentAnswers = answers.filter(item => item[VALUE_POS] !== '').length;
+  const instructions = <AnswerInstructions currentAnswers={currentAnswers} {...props.questionDefinition} {...props} />;
```

Comment:
If both questionDefinition and props define the same setting, this will lead to duplicate props. Does this work instead?

```suggestion
  const instructions = <AnswerInstructions currentAnswers={currentAnswers} {...props.questionDefinition, ...props} />;
```

Reply:
It does not. Duplicate props are not a real problem at this time.